### PR TITLE
[KYUUBI #1354][FOLLOWUP] Add ApiResponse for operation apis

### PR DESCRIPTION
**Pull request by simon824**
### _Why are the changes needed?_
Add ApiResponse for operation apis
#1354

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by ulysses-you — APPROVED** [review]

**Review by cfmcgrady — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala [hunk 1]
```diff
@@ -178,6 +185,13 @@ private[v1] class SessionsResource extends ApiRequestContext {
...
 
+  @ApiResponse(
+    responseCode = "200",
+    content = Array(new Content(
+      mediaType = MediaType.APPLICATION_JSON
+    )),
+    description = "Create an operation with GET_TYPE_INFO type"
+  )
   @POST
   @Path("{sessionHandle}/operations/typeinfo")
```
Comment (cfmcgrady) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala:196]: From the previous discussion, could you update to camel case style?
```scala
@Path("{sessionHandle}/operations/typeInfo")
```

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala [hunk 2]
```diff
@@ -229,6 +264,13 @@ private[v1] class SessionsResource extends ApiRequestContext {
...
 
+  @ApiResponse(
+    responseCode = "200",
+    content = Array(new Content(
+      mediaType = MediaType.APPLICATION_JSON
+    )),
+    description = "Create an operation with GET_TABLE_TYPES type"
+  )
   @POST
   @Path("{sessionHandle}/operations/tabletypes")
```
Comment (cfmcgrady) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala:275]: ditto.

**Review by cfmcgrady — APPROVED** [review]

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala (hunk 1 above)
Comment (ulysses-you) [reply kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala:196]: I have already merged this .. can you send a followup ? thank you !

Comment (simon824) [reply kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/SessionsResource.scala:196]: OK
